napari/_vispy/layers/image.py

```diff
@@ -169,11 +169,22 @@ def _on_depiction_change(self):
     def _on_colormap_change(self):
         self.node.cmap = VispyColormap(*self.layer.colormap)
 
+    def _update_mip_minip_cutoff(self):
+        if isinstance(self.node, VolumeNode):
+            if self.layer.blending != 'opaque':
+                self.node.mip_cutoff = self.node._texture.clim_normalized[0]
+                self.node.minip_cutoff = self.node._texture.clim_normalized[1]
+            else:
+                self.node.mip_cutoff = None
+                self.node.minip_cutoff = None
+
     def _on_contrast_limits_change(self):
         self.node.clim = self.layer.contrast_limits
-        if isinstance(self.node, VolumeNode):
-            self.node.mip_cutoff = self.node._texture.clim_normalized[0]
-            self.node.minip_cutoff = self.node._texture.clim_normalized[1]
+        self._update_mip_minip_cutoff()
+
+    def _on_blending_change(self):
+        super()._on_blending_change()
+        self._update_mip_minip_cutoff()
```

Comment:
could you explain why this order is correct/if it matters?

Reply:
This has to be in this order, cause you need to know the new blending mode before you can tell wether the cutoff can be removed.

Reply:
Would be good to add a comment above the lines to make this clear — I agree with Alister's reaction that it's not obvious when looking at the code.